Makefile

```diff
@@ -263,6 +264,16 @@ generate-modules: ## Run go mod tidy to ensure modules are up to date
 	go mod tidy
 	cd $(TOOLS_DIR); go mod tidy
 
+DOCKER_TEMPLATES := test/e2e/data/addons-helm
+
+.PHONY: generate-e2e-templates
+generate-e2e-templates: $(KUSTOMIZE)
+	$(KUSTOMIZE) build $(DOCKER_TEMPLATES)/v1beta1/cluster-template --load-restrictor LoadRestrictionsNone > $(DOCKER_TEMPLATES)/v1beta1/cluster-template.yaml
+
+.PHONY: generate-flavors ## Generate template flavors.
```

Comment:
I think `## Generate template flavors` should go on the next line (or does this work with `make help`)?